app/src/main/java/com/github/se/eventradar/ui/navigation/NavigationActions.kt

```diff
@@ -41,6 +42,16 @@ class NavigationActions(val navController: NavController) {
   }
 }
 
+fun getTopLevelDestination(targetTextId: Int): TopLevelDestination {
```

Comment:
`Route` is a better/more reliable parameter than the text id, which can change at any time.